source/ranges.tex

```diff
@@ -3333,6 +3362,67 @@
 \end{codeblock}
 \end{itemdescr}
 
+\rSec3[range.owning.view]{Class template \tcode{owning_view}}
+
+\pnum
+\tcode{owning_view} is a move-only view
```

Comment:
I'm not sure a relative clause that long works with the rest of the sentence. This is a note-like introductory sentence, and I feel that the normal English meaning of "move-only" is good enough.

@BRevzin thoughts/ideas?

Reply:
I would definitely prefer "move-only view" to "a view that can be moved but not copied." This is a common-enough pharse. Indeed, the paper that introduced this idea was titled [Move-only views](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1456r1.pdf), and we're also moving `move_only_function`, which... is in the name (and also describes itself as being a "move-only equivalent of `std::function`)

Reply:
If "move-only view" is to be used, I would like to see a definition for it please (and references to that definition added to the use cases).  

We could define "move-only view" via "A \defnadj{move-only}{view} is a \libconcept{view} that may be moved but not copied", but that wording isn't quite right, and we can't just use "view" here because "view" is not a defined term.

Reply:
@tkoeppe, I'd suggest to leave "move-only view" alone.